src/components/MainPage.js

```diff
@@ -5,24 +5,18 @@ import CoinItem from './CoinItem';
 import uuid from 'uuid';
 
 class MainPage extends Component {
-  // coinRender = () => {
-  //   if (!this.props.BTC_rate) {
-  //     return <div>Loading</div>
-  //   } else {
-  //     return <CoinItem
-  //       key={uuid()}
-  //       coinRate={this.props.BTC_rate}
-  //     />
-  //   }
-  // }
-
   render() {
     const { rates } = this.props;
+    if (!rates) {
+      return null;
+    }
     return (
       <div className="box-layout">
         <div className="box-layout__box">
-          <CoinItem key={uuid()} coin={rates.bitcoin.name} rate={rates.bitcoin.price_usd}/>
-          <CoinItem key={uuid()} coin={rates.ethereum.name} rate={rates.ethereum.price_usd}/>
+          {rates.map( (rate) => {
+            return <CoinItem  key={rate.id} coin={rate.name} rate={rate.price_usd}/>
+          })
```

Comment:
Make it as conditional rendering

`{rates && rates.map((rate) => {return <CoinItem  key={rate.id} coin={rate.name} rate={rate.price_usd}/> })`